### PR TITLE
trace-cmd wrapper

### PR DESCRIPTION
**Pull request by matsvanmolle**
Initial commit for the trace-cmd wrapper.

### CODE REVIEW

File: benchkit/commandwrappers/trace-cmd.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Copyright (C) 2023 Huawei Technologies Co., Ltd. All rights reserved.
```
Comment (apaolillo) [inline benchkit/commandwrappers/trace-cmd.py:1]: avoid the dash `-` in module names, it makes the module harder to import. A simple `tracecmd.py` would do.

File: benchkit/commandwrappers/trace-cmd.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+# SPDX-License-Identifier: MIT
+"""
+Command wrapper for the `strace` utility that allows to trace all system calls used in the wrapped
```
Comment (apaolillo) [inline benchkit/commandwrappers/trace-cmd.py:4]: strace?

File: benchkit/commandwrappers/trace-cmd.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+import os
+from typing import List, Optional
+
+from benchkit.commandwrappers import CommandWrapper
+from benchkit.dependencies.packages import PackageDependency
+from benchkit.utils.types import PathType
+
+
+class TraceCmdWrap(CommandWrapper):
```
Comment (apaolillo) [inline benchkit/commandwrappers/trace-cmd.py:16]: perhaps you can re-use some components of the other tracecmd module by importing them. But for now it is sufficiently simple as it is.

**Review by apaolillo — CHANGES_REQUESTED** [review]
Neat! Regarding the report, perhaps a post run hook. However, I'm working a new implementation to avoid the big text file, working on the dat file directly. So for me besides the 2 simple changes, it is good to go.

File: benchkit/commandwrappers/trace-cmd.py (hunk 1 above)
Comment (matsvanmolle) [inline benchkit/commandwrappers/trace-cmd.py:1]: @apaolillo, Should I use the vub copyright?

Comment (apaolillo) [reply benchkit/commandwrappers/trace-cmd.py:1]: indeed, always while you are a VUB student

**Review by apaolillo — APPROVED** [review]

**Pull request merged**